Makefile.accesstokenv1

```diff
@@ -0,0 +1,63 @@
+.PHONY: all pull fetch patch generate clean codegen mod docs move-other patch-post fmt test stage
+
+include Makefile
+
+PACKAGE_NAME=accesstokenv1
+SPEC_BASE_URL:=https://api.swaggerhub.com/apis/equinix-api/accesstoken/1.0
+SPEC_ROOT_FILE:=openapi.yaml
```

Comment:
This is...not true.  I believe I manually downloaded the API info from [Deveoper Portal](https://developer.equinix.com/catalog), unzipped the archive, and then copied the spec file into the spec directory.

This should be fine for now as long as no one tries to sync the spec (CI/CD will only do that in the sync job).

Reply:
The spec is now downloaded from the [Equinix Docs API catalog](https://docs.equinix.com/api-catalog/accesstokenv1).